app/views/claims/pages/privacy.en.html.erb

```diff
@@ -7,10 +7,15 @@
       <h1 class="govuk-heading-l"><%= t(".page_title") %></h1>
 
       <p class="govuk-body">
-        <%= govuk_link_to(
-          "Read privacy information section 8.4",
-          "https://www.gov.uk/government/publications/privacy-information-education-providers-workforce-including-teachers/privacy-information-education-providers-workforce-including-teachers#using-your-data-to-maintain-a-list-of-teachers",
-        ) %> which tells you what we do with you and your mentors’ data when you make contact with us or use this service.
+        Read the <%= govuk_link_to "Privacy information: education providers’ workforce, including teachers (opens in new tab)", "https://www.gov.uk/government/publications/privacy-information-education-providers-workforce-including-teachers/privacy-information-education-providers-workforce-including-teachers", target: "_blank" %>.
```

Comment:
I think the govuk_link_to has an option for new tab, if you want to use it. Not a blocker https://govuk-components.netlify.app/helpers/link/#links-that-open-in-a-new-tab

Reply:
I like it! Will refactor shortly.